.github/workflows/acceptance.yaml

```diff
@@ -0,0 +1,208 @@
+  and openvox-server
+  ${{ (inputs.pre-release-build && inputs.openvox-server-version) ||
+      format(' collection: "{0}", version: "{1}" ',
+             inputs.collection,
+             ((inputs.openvox-server-version == '') && 'latest') ||
+              inputs.openvox-server-version) }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      pre-release-build:
+        description: |-
+          (Pre-release Build) Whether to test unreleased version
+          packages from the artifacts server, or released packages
+          from the given collection.
+
+          If this is true, versions must be valid version, not
+          latest, and collection is ignored. The workflow will
+          download and install the matching openvox package files
+          from the artifacts-url server.
+
+          If this is false, version and collection must match, and the
+          workflow will install the given openvox collection package
+          and then let the system package manager install the latest
+          or version packages from the collection repository.
+        default: true
+        type: boolean
+      openvox-agent-version:
+        description: |-
+          (OpenVox Agent Version) The openvox-agent package version to
+          test. (required if Pre-release Build is true)
+        type: string
+      openvox-server-version:
+        description: |-
+          (OpenVox Server Version) The openvox-server package version
+          to test. (required if Pre-release Build is true)
+        type: string
+      collection:
+        description: |-
+          (Collection) OpenVox collection to use. (ignored if
+          Pre-release Build is true)
+
+          NOTE: This should really only be set to openvox8 when
+          testing the main branch. If you need to test openvox7, you
+          should run the pipeline from the 7.x branch instead.
+        default: 'openvox8'
+        type: string
+      artifacts-url:
+        description: |-
+          URL to the artifacts server. (used if Pre-release Build is
+          true)
+        default: 'https://s3.osuosl.org/openvox-artifacts'
+        type: string
+
+permissions:
+  contents: read
+
+env:
+  RUBY_VERSION: '3.3'
+  # Suppress warnings about Bolt gem versus package use.
+  BOLT_GEM: true
+
+jobs:
+  acceptance:
+    name: Acceptance Tests
+    runs-on: ubuntu-24.04
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - [almalinux, '8']
+          - [almalinux, '9']
+          - [debian, '10']
+          - [debian, '11']
+          - [debian, '12']
+          # No builds of openvox-server in artifacts.voxpupuli.org yet
+          # - [debian, '13', 'amd64', 'daily-latest']
+          - [rocky, '8']
+          - [rocky, '9']
+          - [ubuntu, '18.04']
+          - [ubuntu, '20.04']
+          - [ubuntu, '22.04']
+          - [ubuntu, '24.04']
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Did you set that on purpose? do we need the full history?

Reply:
Removed!